<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clothes_filter</class>
 <widget class="QDialog" name="clothes_filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 229, 199);
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>391</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(53, 66, 89);
</string>
   </property>
   <property name="title">
    <string>پوشاک</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(116, 141, 166);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>نام محصول:     </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>تومان</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(116, 141, 166);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>تا</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(116, 141, 166);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>قیمت :                  </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(116, 141, 166);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>نام فروشنده:   </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>انتخاب کنید</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XXL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XXXL</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>سایز:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>انتخاب کنید</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>بچگانه</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>نوجوان</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>بزرگسال</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>گروه سنی:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>انتخاب کنید</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>مرد</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>زن</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>جنسیت:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>انتخاب کنید</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>تی شرت و پولوشرت</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>پیراهن</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>لباس زیر</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>سوییشرت و هودی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>کاپشن و بارانی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>کت شلوار</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>لباس راحتی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>شلوار جین</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>شلوار کتان</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>شلوار پارچه ای</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>جوراب</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>اکسسوری</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>پلیور و ژاکت</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>کیف و کفش</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>دسته بندی: </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(116, 141, 166);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string>کلمات کلیدی:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>89</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
   </property>
   <property name="text">
    <string>حذف فیلتر</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>87</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(53, 66, 89);
font: 11pt &quot;Lalezar&quot;;</string>
   </property>
   <property name="text">
    <string>فیلتر کردن</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
